config/connection.php

```diff
@@ -6,4 +6,4 @@
   $q_string = "USE serverus;";
   mysqli_query($dbc, $q_string);
   
-?>
+?>
```

Comment:
Didn't mean to commit this but it's literally a blank line removed, nothing is changed.